source/uwp/Renderer/lib/AdaptiveExecuteAction.h

```diff
@@ -90,6 +90,15 @@ namespace AdaptiveNamespace
             return AdaptiveActionElementBase::put_Tooltip(tooltip);
         }
 
+        IFACEMETHODIMP put_IsEnabled(boolean isEnabled) override
```

Comment:
definitely not something to do in this PR, but I think if we refactor our classes using cppwinrt, we should be able to derive classes in a way that makes it so we can get out of the business of manually calling the base (`LinkButton` does something similar to this).

guessing it might require breaking API surface changes, so maybe part of our next major rev? just a thought :)

Reply:
I agree, I'd love to get more cppwinrt in here. I feel like it shouldn't require breaking changes? But I don't know. Definitely something to work towards at some point.